app/main.py

```diff
@@ -1 +1,34 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(self, name: str, health: int = 100) -> None:
+        self.name = name
+        self.health = health
+        self.hidden = False
+        Animal.alive.append(self)
+
+    def __repr__(self) -> str:
+        return f"{{" \
+            f"Name: {self.name}," \
+            f" Health: {self.health}," \
+            f" Hidden: {self.hidden}" \
+            f"}}"
+
+
+class Herbivore(Animal):
+    def hide(self) -> None:
+        if self.hidden:
+            self.hidden = False
+        else:
+            self.hidden = True
```

Comment:
<img width="870" alt="image" src="https://github.com/mate-academy/py-herbivores-and-carnivores/assets/93856423/5ff5b4be-1616-49cb-9e64-f2bf127cc290">